Typo in helm delete for the publish artefact action

### Description
There is a typo in the helm delete step for the publish artefact action which tries to delete a chart using different variables with which it installed the chart with in the first place. This in turn produces the error below:
```
##[error]Process completed with exit code 1.
```

Make sure that the namespaces for creating charts is the same for deleting charts.

### Acceptance criteria
- [ ] Artefacts are deleted without error-ing

### Related issues
-
-
-

--- a/.github/workflows/publish-artefact.yaml
+++ b/.github/workflows/publish-artefact.yaml
@@ -93,7 +93,7 @@ jobs:
         aws s3 cp ./repo/index.yaml s3://${S3_BUCKET_NAME} --metadata-directive REPLACE --cache-control max-age=0,no-cache,no-store,must-revalidate --content-type "text/plain"
 
     - name: Delete the chart
-      run: helm delete $GITHUB_HEAD_REF-$SHORT_SHA --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
+      run: helm delete $CHART_NAME-$SHORT_SHA --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
 
     - name: Check if the namespace exists
       run: echo '::set-env name=NAMESPACE_COUNT::'$(kubectl get namespaces -A | grep '$ACTOR_LOWERCASE-$GITHUB_RUN_ID ' | wc -l)

--- a/stable/autonity/CHANGELOG.md
+++ b/stable/autonity/CHANGELOG.md
@@ -4,6 +4,10 @@ All notable changes to this chart will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.1]
+- Disable freezer functionality with deployment of autontity hotfix 0.4.1
+- Support clearmatics/autonity-init v1.0.8
+
 ## [1.2.0]
 - Support helm v3.2.4 [issue #1](https://github.com/clearmatics/charts-ose-helm3/issues/1)
 - Support clearmatics/autonity v0.4.0 [issue #2](https://github.com/clearmatics/charts-ose-helm3/issues/2)

--- a/stable/autonity/Chart.yaml
+++ b/stable/autonity/Chart.yaml
@@ -1,6 +1,6 @@
 name: autonity
 apiVersion: v1
-version: 1.2.0
+version: 1.2.1
 kubeVersion: ">=1.15.11-0"
 description: Deploy single Autonity instance
 keywords:
@@ -12,4 +12,4 @@ sources:
 maintainers:
 - name: eastata
   email: aleksandr.poliakov@clearmatics.com
-appVersion: v0.4.0
+appVersion: v0.4.1

--- a/stable/autonity/values.yaml
+++ b/stable/autonity/values.yaml
@@ -2,11 +2,11 @@
 
 imageAutonity:
   repository: clearmatics/autonity
-  tag: v0.4.0
+  tag: v0.4.1
 
 imageAutonity_init:
   repository: clearmatics/autonity-init
-  tag: v1.0.7
+  tag: v1.0.8
 
 image_eth_keys_generator:
   repository: clearmatics/eth-keys-generator

